serverless-http.js

```diff
@@ -16,6 +16,10 @@ module.exports = function(app, opts) {
   const options = Object.assign({}, defaultOptions, opts);
 
   return (evt, ctx, callback) => {
+    if (options.callbackWaitsForEmptyEventLoop != null) {
```

Comment:
Hi, I thought I left another comment here a while ago but can't find it.
Thanks for explaining - I see there are situations in which you don't want to wait for the event loop to clear on every request.
However, there is no guarantee that a frozen container will ever be thawed - it might just as well be destroyed.
So if you need guarantee that certain events are handled, you must wait for the event loop to clear.

Therefore I'd like to propose to make it possible to set callbackWaitsForEmptyEventLoop via an option, as was in the original PR.
Would you consider doing so? If you want I can make a new PR.